test/functional/tests/failover_standby/test_failover_multihost.py

```diff
@@ -0,0 +1,534 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+from time import sleep
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import (
+    SeqCutOffPolicy,
+    CacheMode,
+    CleaningPolicy,
+    CacheLineSize,
+    CacheStatus,
+)
+from api.cas.casadm_parser import get_caches
+from api.cas.init_config import InitConfig
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet
+from storage_devices.drbd import Drbd
+from storage_devices.raid import Raid, RaidConfiguration, MetadataVariant, Level
+from test_tools.dd import Dd
+from test_tools.drbdadm import Drbdadm
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite
+from test_tools.fs_utils import readlink
+from test_utils.drbd import Resource, Node
+from test_utils.os_utils import sync, Udev
+from test_utils.size import Size, Unit
+
+
+cache_id = 5
+raid_size = Size(1, Unit.GibiByte)
+core_size = Size(500, Unit.MebiByte)
+metadata_size = Size(100, Unit.MebiByte)
+cache_exp_obj_path = f"/dev/cas-cache-{cache_id}"
+cls = CacheLineSize.LINE_32KiB
+
+
+@pytest.mark.require_disk("metadata_dev", DiskTypeSet([DiskType.nand]))
+@pytest.mark.require_disk("core_dev", DiskTypeSet([DiskType.hdd]))
+@pytest.mark.require_disk("raid_dev1", DiskTypeSet([DiskType.optane]))
+@pytest.mark.require_disk("raid_dev2", DiskTypeSet([DiskType.optane]))
+@pytest.mark.multidut(2)
+@pytest.mark.require_plugin("power_control")
+def test_functional_activate_twice_round_trip():
```

Comment:
I suggest to parametrize two both tests to have a version with and without file system - this generates significantly different I/O pattern to CAS and potential debugging should be easier on a version without file system